### ISSUE TITLE
[Accessibility] Line-by-line audio cues are not played when column position is away from the event trigger spot

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#174857]
CC @meganrogge

## Reproducible Steps

1. Load screen reader.

1. Open the following in code-insiders mode: https://github.dev/microsoft/vscode/blob/main/README.md

1. Press Ctrl+G (go to line) and type `64,9` and hit enter.

1. While keeping the column position at 9, move lines by pressing UpArrow and DownArrow and come back to line 64: audio cue is not played.

1. This time, place your cursor to the left (`64,8`) and do the same thing as above: audio cue is now played.

## Current Behavior

1. Line 64 has a warning and the audio cue is not played when you press UpArrow/DownArrow and come back to this line.

## Expected Beaavior

Audio cues should be played when UpArrow/DownArrow is used.

## System Info

Version: 1.76.0-insider (system setup)
Commit: 78cb4eb690930964f4dcddde7c4828bdb5beaabd
Date: 2023-02-20T11:45:27.892Z
Electron: 19.1.10
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by dbaeumer** [comment]
@meganrogge tried to verified this but failed. Here is what I did:

- Windows version
- NVDA installed
- Screen Reader Optimized enabled
- Followed the steps from above but no audio cues on line move

I also looked in the settings and there is no setting for audio cues on line move. Anything I am missing?

**Comment by meganrogge** [comment]
`audioCues.lineHasError` is what you need @dbaeumer

**Comment by meganrogge** [comment]
and then move the cursor to an error line

**Comment by bhavyaus** [comment]
Verified that the audio cue plays on error. I wasn't able to see a warning in the file mentioned in the repro step so I made one up.
